reconcile-tags/reconcile-tags.py

```diff
@@ -0,0 +1,32 @@
+"""Reads a json file mapping docker digests to tags and reconciles them.
+
+Reads tap-map.json, currently a file in the same folder as this script
+and parses the json. Calls gcloud beta container images add-tag on each entry.
+If there are no changes that api call is no-op.
+"""
+
+import glob
+import json
+from subprocess import call
+
+
+def main():
+    files = glob.glob('./*.json')
+    for f in files:
+        print f
```

Comment:
Could you turn these prints into logging calls, and log a little bit more metadata?